.pylintrc

```diff
@@ -40,7 +40,7 @@ confidence=
 
 # disable=all
 enable=all
-disable=C0114,C0115
+disable=C0114,C0115,C0330,invalid-name,suppressed-message
```

Comment:
Instead of disabling all invalid-name checking you could also add names that you like that are disabled by default to the accepted/good list.

Reply:
@simbuerg this one is not resolved, just get's shown as outdated

Reply:
Ah right, missed it.

Reply:
Fixed for now. The list is too long for my taste, but we can trim it later...